test/JDBC/input/operator_binary_whitespace-vu-verify.sql

```diff
@@ -0,0 +1,139 @@
+-- bit AND/OR/XOR
+select 99&-5

```

Comment:
Would you mind add a test case to put multiple operator expressions into single select ? Like : 
```
select 99&-5, 99&-5, 99&-5 , 99&-5, 99&-5, 99&-5, 99&-5 , a, b, c from test 
```

Reply:
testcases added